[2022-11-12T13:00:39.120+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ETL.Load scheduled__2022-11-11T00:00:00+00:00 [queued]>
[2022-11-12T13:00:39.165+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ETL.Load scheduled__2022-11-11T00:00:00+00:00 [queued]>
[2022-11-12T13:00:39.168+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-12T13:00:39.177+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-12T13:00:39.179+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-12T13:00:39.260+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): Load> on 2022-11-11 00:00:00+00:00
[2022-11-12T13:00:39.280+0000] {standard_task_runner.py:55} INFO - Started process 232 to run task
[2022-11-12T13:00:39.301+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'Load', 'scheduled__2022-11-11T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmptasjh677']
[2022-11-12T13:00:39.311+0000] {standard_task_runner.py:83} INFO - Job 4: Subtask Load
[2022-11-12T13:00:39.582+0000] {task_command.py:376} INFO - Running <TaskInstance: ETL.Load scheduled__2022-11-11T00:00:00+00:00 [running]> on host bc6023eb63d2
[2022-11-12T13:00:40.203+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL
AIRFLOW_CTX_TASK_ID=Load
AIRFLOW_CTX_EXECUTION_DATE=2022-11-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-11T00:00:00+00:00
[2022-11-12T13:00:40.216+0000] {logging_mixin.py:120} INFO - Prueba Load EXITOSO!
[2022-11-12T13:00:40.238+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-12T13:00:40.287+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=Load, execution_date=20221111T000000, start_date=20221112T130039, end_date=20221112T130040
[2022-11-12T13:00:40.514+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-12T13:00:40.624+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-11-12T14:08:07.384+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ETL.Load scheduled__2022-11-11T00:00:00+00:00 [queued]>
[2022-11-12T14:08:07.466+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ETL.Load scheduled__2022-11-11T00:00:00+00:00 [queued]>
[2022-11-12T14:08:07.478+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-12T14:08:07.491+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-12T14:08:07.561+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-12T14:08:07.905+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): Load> on 2022-11-11 00:00:00+00:00
[2022-11-12T14:08:07.983+0000] {standard_task_runner.py:55} INFO - Started process 237 to run task
[2022-11-12T14:08:08.017+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'Load', 'scheduled__2022-11-11T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/DAG_etl_testDef.py', '--cfg-path', '/tmp/tmph4ckfaac']
[2022-11-12T14:08:08.043+0000] {standard_task_runner.py:83} INFO - Job 7: Subtask Load
[2022-11-12T14:08:09.210+0000] {task_command.py:376} INFO - Running <TaskInstance: ETL.Load scheduled__2022-11-11T00:00:00+00:00 [running]> on host 5e1e1b90c781
[2022-11-12T14:08:11.298+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL
AIRFLOW_CTX_TASK_ID=Load
AIRFLOW_CTX_EXECUTION_DATE=2022-11-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-11T00:00:00+00:00
[2022-11-12T14:08:11.311+0000] {logging_mixin.py:120} INFO - Prueba Load EXITOSO!
[2022-11-12T14:08:11.320+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-12T14:08:11.433+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=Load, execution_date=20221111T000000, start_date=20221112T140807, end_date=20221112T140811
[2022-11-12T14:08:11.643+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-12T14:08:12.048+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
